### PR TITLE
added trello module for issue #99

### PR DESCRIPTION
**Pull request by vikas-kundu**
Here is a gif of the module: https://asciinema.org/a/erbXUaPUTxS9WeYyk99X1a37p

### CODE REVIEW

**Comment by z3y50n** [comment]
Everything looks good to me.
@saeeddhqan I think you can merge, maybe you can check the regexes if you want any changes

**Comment by vikas-kundu** [comment]
Just to elaborate more on the regexs, the usernames associated with a keyword are found inside brackets and contain the keyword i.e. for the word foo, usernames may be (foo123), (foobar), (barfoo) etc. 
The handles on the other hand start with an @ symbol and may contain any word. Some handles like @keyframes are not valid so they have been added in a filter blacklist.

File: modules/search/trello.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		'default_q': f"site:trello.com {query}",
+		'yippy_q': f'"trello.com" {query}',
+		'millionshort_q': f'site:trello.com "{query}"',
+		'qwant_q': f'site:trello.com {query}'
+	}
+
+	self.thread(search, self.options['thread'], engines, query, q_formats, limit, count, meta['sources'])
+	
+	output['links'] = list(self.reglib().filter(r"https?://([\w\-\.]+\.)?trello\.com/", list(set(LINKS))))
+	output['usernames'] = [ i for i in set(re.findall(r"\([\w]{3,}\)", PAGES)) if query in i  ]
```
Comment (saeeddhqan) [inline modules/search/trello.py]: Check this regex:
`\(([\w]{3,})\)`

Comment (vikas-kundu) [reply modules/search/trello.py]: @saeeddhqan  usernames are present on search engine pages in this format (see image: https://ibb.co/TKYjPV1) . So it has to be within curly braces. So \(.*\). Now that it is of type word so \([\w]\). Now to more refine the search, i filter such usernames which do no contain the search query. The {3,} part is to make sure at least it is of length 3 or more. I can remove this part but the rest seems fine to me.

Comment (saeeddhqan) [reply modules/search/trello.py]: It's not good to show usernames like this: (@abc)\n(@cba)

Comment (vikas-kundu) [reply modules/search/trello.py]: @saeeddhqan ok so should i remove the usernames and handles part  ?

Comment (saeeddhqan) [reply modules/search/trello.py]: I mean, this was?

Comment (vikas-kundu) [reply modules/search/trello.py]: There are three parts to the output
1. Links: Normal trello board links
2. usernames: The ones inside round braces (vikas123)
3. handles: Those with mentions on public trello boards i.e. @trello123, @something
So, if i am understanding it, I need to remove no. 2 and no. 3 part. Right?

Comment (saeeddhqan) [reply modules/search/trello.py]: > It's not good to show usernames like this: (@abc)\n(@cba)
It didn't mean to remove the usernames. it means, isn't it better to show usernames like this: @abc\n@cba
shall we?

Comment (vikas-kundu) [reply modules/search/trello.py]: Oh. Got it. Will make the output neat.

Comment (saeeddhqan) [reply modules/search/trello.py]: I think this regex \(([\w]{3,})\) will find what we want.

Comment (vikas-kundu) [reply modules/search/trello.py]: A. https://ibb.co/jgt4Z5T
               or 
B. https://ibb.co/B2nFtxZ
?

Comment (saeeddhqan) [reply modules/search/trello.py]: A

Comment (vikas-kundu) [reply modules/search/trello.py]: Made the necessary changes https://github.com/saeeddhqan/Maryam/pull/100/commits/2a6e367a979564889a740319303763e6d986b9eb

**Pull request merged**